Makefile

```
@@ -340,3 +340,7 @@ clean-operator: ## Clean Operator
 	oc delete accounts --all -n ${NAMESPACE}
 	oc delete awsfederatedaccountaccess --all -n ${NAMESPACE}
 	oc delete awsfederatedrole --all -n ${NAMESPACE}
+
+.PHONY: op-generate
+op-generate: ## Generate CRDs and k8s
+  ./generate_crds.sh
```

This needs to be tab
```suggestion
	./generate_crds.sh
```